controls/SkyView_RTS.js

```diff
@@ -52,6 +57,8 @@ module.exports = function(camera, scene, domElement, loadObjects){
 
         var canvasBoundingRect = domElement.getBoundingClientRect();
 
+        mousePos = {x: e.clientX, y: e.clientY};
```

Comment:
Instead of creating a fresh object  at each mouse move event (which is often), we might want to save the event object:
````js
var lastMouseMoveEvent; // instead of mousePos

    // afterwards
    lastMouseMoveEvent = e;

// and use clientX/clientY at the end
````